lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1733,6 +1733,65 @@ Error ONNXModelLoader::loadArgMax(const ONNX_NAMESPACE::NodeProto &op,
   return Error::success();
 }
 
+Error ONNXModelLoader::loadUpsample(const ONNX_NAMESPACE::NodeProto &op,
+                                    ArgumentDictionaryTy &dict) {
+
+  RETURN_ERR_IF_NOT(opsetVersion_ < 10,
+                    "Upsample Operator is deprecated for Opset 10 and above");
+
+  const std::string &opName = loadOperatorName(op);
+  NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getNodeValueByName(op.input(0)));
+  std::string mode;
+  if (dict.count("mode")) {
+    ASSIGN_VALUE_OR_RETURN_ERR(mode, loadStr(dict.at("mode")));
+  }
+
+  /// Only Nearest Mode is supported
+  RETURN_ERR_IF_NOT(mode.compare("nearest") == 0,
+                    "Upsample Operator has nearest mode support only");
+
+  /// Scale is always float as per onnx documentation
+  std::vector<float> scales;
+
+  RETURN_ERR_IF_NOT(opsetVersion_ > 6,
```

Comment:
Move this to the top/merge with the check for `opsetVersion_ < 10`

Reply:
Done.